### PR TITLE
Include field name in error message when field value proc eval raises and error

### PR DESCRIPTION
**Pull request by juleea**

### CODE REVIEW

File: fixy.gemspec [hunk 1]
```diff
@@ -18,6 +18,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_development_dependency 'rspec'
+  spec.add_development_dependency 'pry-byebug'
```
Comment (alexgerstein) [inline fixy.gemspec:21]: Remove.

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 require 'rspec'
 require 'fixy'
+require 'pry'
```
Comment (alexgerstein) [inline spec/spec_helper.rb:3]: Remove?

File: spec/fixy/record_spec.rb [hunk 3]
```diff
@@ -217,6 +217,18 @@ class PersonRecordWithLineEnding < Fixy::Record
       PersonRecordWithLineEnding.new.generate.should eq('Use My Value'.ljust(20) << "\r\n")
     end
   end
+
+  context 'when the value proc raises and error' do
```
Comment (alexgerstein) [inline spec/fixy/record_spec.rb:221]: an*

**Comment by alexgerstein** [comment]
Looks good! @zp-alfred Accept!

**Comment by zp-alfred** [comment]
Accepted by alexgerstein!

**Pull request closed without merging**